Translations/en.xliff

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Can we move the translation related changes to a separate pull request? I'd like to have only issue 3093 tackled here.